app/status/views.py

```diff
@@ -6,4 +7,11 @@
 
 @status.route('/_status')
 def status_no_db():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    try:
+        return jsonify(status="ok", app_version=utils.get_version_label(),
+                       db_version=utils.get_db_version())
+
+    except SQLAlchemyError:
+        current_app.logger.exception('Cannot connect to database.')
+        return jsonify(status="error", message="Database is down"), 500
```

Comment:
I was sort of thinking the error messages should just be as simplistic as possible.

[Here](https://github.com/alphagov/digitalmarketplace-admin-frontend/pull/42/files), I'm returning a similarly pared-down message for the admin-frontend app. 

Is there a conceivable advantage to having version numbers in all our error messages?
And then, in the case we include one here, do we want the admin-frontend app to know about the API app_version number if it gets an error?